src/main/resources/db/migration/V1_57__more_document_storage.sql

```diff
@@ -0,0 +1,3 @@
+Alter table document_location add column category varchar(100) default null;
+Alter table document_location add column original_document_file_name varchar(256) default null;
```

Comment:
these are non-nullable on the entity. Should either be nullable there or non-nullable here. Non nullable here would be better

Reply:
Done